task.py

```diff
@@ -420,6 +436,54 @@ def create_node_port_service(self, nodeport: int) -> str:
             "get service tft-nodeport-service -o=jsonpath='{.spec.clusterIP}'"
         ).out
 
+    def create_ingress_multi_network_policy(self, ingressPort: int):
+        in_file_template = "./manifests/allow-ingress-mnp.yaml.j2"
+        out_file_yaml = "./manifests/yamls/allow-ingress-mnp.yaml"
+
+        template_args = {
+            **self.get_template_args(),
+            "ingress_port": f"{ingressPort}",
+            "secondary_network_nad": self.ts.connection.secondary_network_nad,
+        }
+
+        self.render_file(
+            "Ingress Multi Network Policy",
+            in_file_template,
+            out_file_yaml,
+            template_args,
+        )
+        r = self.run_oc(f"apply -f {out_file_yaml}", may_fail=True)
+        if r.returncode != 0:
+            if "already exists" not in r.err:
+                logger.info(r)
+                sys.exit(-1)
+
+        return self.run_oc("get multi-networkpolicies allow-ingress-mnp").out
```

Comment:
The reason why I added this check here instead of adding `die_on_error=True` is because if the resource already exists then it might fail even though it shouldn't

Reply:
I am talking about the `run_oc("get...")`. That one should not fail, as we just created the resource.

If you would anticipate it to fail (which you don't need too), then you also should handle errors somehow. `die_on_error=True` is a IMO suitable way to handle it here.